### PR TITLE
bugfix/15863-remove-hide-by-translation

### LINKED ISSUE #15863: convert SVG in PowerPoint to shapes for Polar Chart is very large and off the chart
We are implementing the new highcharts and something we use is to convert SVG in PowerPoint to shapes.
This works for most Highchart graphs but we have an issue with the polar chart. 

#### Live demo with steps to reproduce
https://www.highcharts.com/demo/polar
Export to SVG and paste in PowerPoint and convert to Shapes but then it looks not correct and one shape is too big.

#### Product version
Highcharts 9.1.1

### PR DESCRIPTION
**Pull request by pawellysy**
Part of #15863, the scrollbar is no longer initially rendered with `y: -9999`.

### CODE REVIEW

File: ts/Core/Scrollbar.ts [hunk 1]
```diff
@@ -596,6 +596,10 @@ class Scrollbar {
         let xOffset = height,
             yOffset = 0;
 
+        // Make the scrollbar visible when it is repositioned, #15763.
+        scroller.group.attr({
+            visibility: 'visible'
```
Comment (TorsteinHonsi) [inline ts/Core/Scrollbar.ts:601]: Use the shorter `.show()` function.

File: ts/Core/Scrollbar.ts [hunk 2]
```diff
@@ -661,8 +665,8 @@ class Scrollbar {
             size = scroller.size,
             styledMode = scroller.chart.styledMode,
             group = renderer.g('scrollbar').attr({
-                zIndex: options.zIndex,
-                translateY: -99999
+                visibility: 'hidden',
```
Comment (TorsteinHonsi) [inline ts/Core/Scrollbar.ts:668]: In this case too it will probably be better to use the shorthand `.hide()` function. It is chainable.

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]
Thanks! Remarks regarding the `show` and `hide` shorthand functions.

**Review by pawelfus — APPROVED** [review]
Thanks!

File: ts/Core/Scrollbar.ts [hunk 3]
```diff
@@ -660,10 +662,12 @@ class Scrollbar {
...
-            group = renderer.g('scrollbar').attr({
-                zIndex: options.zIndex,
-                translateY: -99999
-            }).add();
+            group = renderer.g('scrollbar')
+                .attr({
+                    zIndex: options.zIndex
+                })
+                .add()
+                .hide(); // initially hide the scrollbar #15863
```
Comment (TorsteinHonsi) [inline ts/Core/Scrollbar.ts:670]: If possible, I think it is better to hide it before add. That would prevent rendering and a potential flash of visible content.

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]

**Review by TorsteinHonsi — APPROVED** [review]

**Pull request merged**